### PR TITLE
CONCF-552 Categories as Curated Content sections

### PR DESCRIPTION
**Pull request by rogatty**
https://wikia-inc.atlassian.net/browse/CONCF-552

### CODE REVIEW

File: front/templates/main/components/curated-content-item.hbs [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
 	<figcaption class='item-figcaption'>
 		<div class='item-caption-row'>
-		<span class='item-caption label'>{{label}}</span>
-		<span class='item-caption grid-icon'>
-			<div class='grid-icon-container'>
-				{{svg 'category-thumb-icon' role='img' class='category-icon' viewBox='0 0 17 17'}}
+			<span class='item-caption label'>{{model.label}}</span>
+			<div class='item-caption grid-icon'>
+				<div class='grid-icon-container'>
+					{{#if isArticle}}
```
Comment (hakubo) [inline front/templates/main/components/curated-content-item.hbs]: how about adding a property
'iconName' or something like that
so we could
`{{svg (iconName) role='img' viewBox='0 0 17 17'}}`
would that work?

File: front/scripts/main/components/CuratedContentComponent.ts [hunk 2]
```diff
@@ -1,38 +1,100 @@
 /// <reference path="../app.ts" />
 ///<reference path="../mixins/LoadingSpinnerMixin.ts"/>
 ///<reference path="../mixins/TrackClickMixin.ts"/>
+///<reference path="../models/CuratedContentModel.ts"/>
 'use strict';
 
 App.CuratedContentComponent = Em.Component.extend(App.LoadingSpinnerMixin, App.TrackClickMixin, {
```
Comment (hakubo) [inline front/scripts/main/components/CuratedContentComponent.ts]: Could you add some unit tests?
That would be :100: points

**Comment by hakubo** [comment]
Looking good! Thanks

File: front/templates/main/components/curated-content-item.hbs (hunk 1 above)
Comment (rogatty) [inline front/templates/main/components/curated-content-item.hbs]: I've tried, it doesn't work. Surely it's possible but I'd have to modify `svgHelper` and I don't want to do this now - deadlines and all ;)

File: front/scripts/main/components/CuratedContentComponent.ts (hunk 2 above)
Comment (rogatty) [inline front/scripts/main/components/CuratedContentComponent.ts]: Good idea but how about we do this after our little experiment succeeds?

File: front/templates/main/components/curated-content-item.hbs (hunk 1 above)
Comment (hakubo) [inline front/templates/main/components/curated-content-item.hbs]: mkay

File: front/scripts/main/components/CuratedContentComponent.ts (hunk 2 above)
Comment (hakubo) [inline front/scripts/main/components/CuratedContentComponent.ts]: We are already working on stuff that are nice to haves, 
I think we do have time for some unit tests :)

File: front/templates/main/components/curated-content-item.hbs (hunk 1 above)
Comment (vforge) [inline front/templates/main/components/curated-content-item.hbs]: @rogatty can you create ticket for modifying `svgHelper` then?

File: front/scripts/main/components/CuratedContentComponent.ts (hunk 2 above)
Comment (vforge) [inline front/scripts/main/components/CuratedContentComponent.ts]: @hakubo what should we postpone and wait for unit tests? can we QA in the meantime?

Comment (hakubo) [inline front/scripts/main/components/CuratedContentComponent.ts]: I'd wait for unit test, they might lead to some refactors and then we would have to test it again

**Pull request merged**